npm/src/lib/libDef.js

```diff
@@ -148,7 +148,8 @@ export async function getLocalLibDefs(
   return libDefs;
 };
 
-const FLOW_DIR_NAME_RE = /^flow-(all|([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x))$/;
+const FLOW_DIR_NAME_RE =
+  /^flow-(all|([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x)(_([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x))?)$/
```

Comment:
Hehe, oh regexp.

nit: Maybe breaking this down with strings would be a little more readable?

```javascript
const FLOW_VER = '([><]?=)?v([0-9]+)\.([0-9]+|x)\.([0-9]+|x)';
const FLOW_DIR_NAME_RE = new RegExp(
  `^flow-(all|${FLOW_VER_RANGE}(_${FLOW_VER_RANGE})?)$`
);
```